src/Illuminate/Foundation/helpers.php

```diff
@@ -470,6 +470,27 @@ function factory()
     }
 }
 
+if (! function_exists('has_errors')) {
+    /**
+     * Returns true if session has errors, and optionally key-specific errors.
+     *
+     * @param  string|null  $key
+     * @param  string|null  $bag
+     * @return bool
+     */
+    function has_errors($key = null, $bag = 'default')
```

Comment:
Thanks for pointing out this, indeed it should be consistent.

It's pretty standard to pass `null` for using the default value, so one doesn't have to remember if it's "default", "\_default", "\_\_default\_\_" or whatever. But indeed this is a different changeset.